.github/workflows/deploy.yml

```diff
@@ -14,7 +14,8 @@ env:
 jobs:
   Integration_And_Deployment:
     runs-on: ubuntu-latest
-
+    env:
```

Comment:
There's probably a [better way we can do this](https://stackoverflow.com/questions/72387730/github-action-how-to-check-if-pr-creator-is-part-of-a-specific-team)? Though, not something I'm _super_ concerned about immediately?

Reply:
That was the recommended way since secrets are not available during if statements 🙃